### PR TITLE
OVERVIEW: Can now retrieve trips in JSON

### PR DESCRIPTION
**Pull request by James-Firth**
app.js - removed line that was causing errors (reason unknown, sorry)
secret.js - using a different DB now (named "barbosa") to make it clear what DB we're using.
styles.css - SHA hash and file length changed automagically, committing to be safe.
Trip.js - removed extra whitespace.
trip.js - Removed plurarlization so the comments were actually correct.
        - showTrip function now returns a JSON string with the trip info when we go to /trip/:id
        - trip actually saves to the mongodb collection and returns with a JSON string that includes the tripID

### CODE REVIEW

File: controllers/trip.js [hunk 1]
```diff
@@ -15,20 +15,26 @@ exports.load = function(req, res, next, id){
...
  * Get a trip!
  */
 
 exports.showTrip = function(req, res) {
-  trip = req.trip
-  res.render('trip/show', {
-    title: trip.name + " " + trip.id,
-    trip: trip
-  })
+  var ObjectId = mongoose.Types.ObjectId;
```
Comment (willvoutier) [inline controllers/trip.js:23]: What's this for? We don't use it anywhere in the function?

File: controllers/trip.js [hunk 2]
```diff
@@ -15,20 +15,26 @@ exports.load = function(req, res, next, id){
...
+                    status: 500
+                });
+            } else {
+                res.json(trips);
+            }
+        });
 };
 
 /**
- * GET /trips/create
```
Comment (willvoutier) [inline controllers/trip.js:31]: I don't know that this is a good change... Generally, we use the plural form of the word. http://stackoverflow.com/questions/6845772/rest-uri-convention-singular-or-plural-name-of-resource-while-creating-it

**Comment by willvoutier** [comment]
^ Put a few comments ^. Also, good thing you got this working but we should delete some code in models/trips.js if we are not using the param thing. I think there's a load() function or something.

File: controllers/trip.js (hunk 1 above)
Comment (James-Firth) [inline controllers/trip.js:23]: Hmm, must be leftover code... I'll make some changes today and do another pull request?

File: controllers/trip.js (hunk 2 above)
Comment (James-Firth) [inline controllers/trip.js:31]: Ah alright, I just changed the comments to be standardized with the actual routing, I forget if I changed the routes or not, I just noticed the docs didn't match. I'll also switch those back when I do the cleanup to remove unused code.

File: controllers/trip.js (hunk 1 above)
Comment (willvoutier) [inline controllers/trip.js:23]: Naw. One review should be enough unless you want someone else to take a look or you're really unsure.

**Pull request closed without merging**